extension/views.py

```diff
@@ -224,6 +224,26 @@ def get_friends(request):
                          'type': 'contact'})
             if len(data) > 5:
                 break
+            
+    url = request.GET.get('url', '')
+    
+    messages = ChatMessage.objects.filter(url=url).order_by('-date').select_related()
+    
+    id_list = []
+    for suggested_user in data:
```

Comment:
why not just create id_list above and add an append inside the for loop where items get added to data?